capsules/src/i2c_master_slave_driver.rs

```diff
@@ -18,6 +18,9 @@ use kernel::ReturnCode;
 use kernel::common::take_cell::{MapCell, TakeCell};
 use kernel::hil;
 
+// Syscall driver number.
+pub const DRIVER_NUM: usize = 0x20003;
```

Comment:
Actually, on second thought, I'm not sure that this one is right. The docs say
> doc/syscalls/README.md:|   | 0x20003       | I2C Master       | Raw I2C Master interface                   |
but this is a mixed master/slave thing.

Can you pull this one out? #761 gives this driver a temporary number while we sort out what exactly we want to do here.

Reply:
You're right - I missed that, and I can pull it out

Neither of the documented I2C drivers are used anywhere in tock though, so we should think about how to fix this, either by separating the current driver into master/slave pieces or committing to them being combined.